src/main/java/net/mcreator/element/converter/v2022_1/FoodToItemConverter.java

```diff
@@ -59,7 +61,8 @@ public GeneratableElement convert(Workspace workspace, GeneratableElement input,
 			if (food.get("specialInfo") != null)
 				food.getAsJsonArray("specialInfo").iterator()
 						.forEachRemaining(element -> specialInfo.add(element.getAsString()));
-			item.specialInfo = specialInfo;
+			item.specialInformation = new StringProcedure(null,

```

Comment:
You should use StringUtils.splitCommaSeparatedStringListWithEscapes, not a manual approach

Reply:
The method `splitCommaSeparatedStringListWithEscapes` returns a list of string, but I only need one string

Reply:
I see. But why do you additionally escape commas? Commas that are not part of string list are already escaped by the users

Reply:
Because `specialinfo` is an ArrayList, if there's a comma (`,`) in the text, it means it's a comma we want to "keep" (not one that means it's supposed to be a new line. As you can see in the help tip :
>  Use the backslash with a comma after ("\\,") to insert comma in the text instead of separating entries.

So I replace this simple comma (`,`) with a backlash comma (`\\,`)

Reply:
Commas inside string inside list are already escaped as list is not used for splitting to my knowledge, or am I mistaken?

Reply:
Nevermind, I see your point now looking at the code, sorry for the confusion